warehouse/query-core/src/test/java/datawave/query/function/JexlEvaluationTest.java

```diff
@@ -159,7 +159,7 @@ public void testContentPhraseFunction() {
         boolean foundPhrase = false;
         Attributes attrs = (Attributes) d.get("HIT_TERM");
         for (Attribute<?> attr : attrs.getAttributes()) {
-            if (attr.getData().equals("TOKFIELD:big red dog")) {
+            if (attr.toString().equals("TOKFIELD:big red dog")) {
```

Comment:
We should not be counting on toString() here.  I think we still need to verify that getData() returns this value.

Reply:
getData() returns an Object, the delegate, which is actually the underlying Type (inside the TypeAttribute)
In this case, it returns a HitTermType which extends StringType which extends BaseType<String>...
I could call attr.getData().toString() and fail if getData() returns null, 
but our TypeAttribute has a toString() that checks for a null delegate and then either returns getDelegateAsString, or is returns the class name and other stuff. That null case return won't match what's in the expected results so the test would still work.
Thoughts?

Reply:
indeed, Content getData() returned the Content's data, which is always a String but returned as an Object.

Reply:
also, these are only unit tests, not production code, that get the string values to compare against expected string values.

Reply:
If one would happen to change what toString() returns, that should not break test cases.  If one changes for getData() returns, one could expect that to break test cases.

Reply:
I'll avoid the toString and just do something like this:

// if its a TypeAttribute
((TypeAttribute)attr).getType().getDelegateAsString()

Reply:
I made changes and pushed to the branch.